locale/en/knowledge/getting-started/what-is-require.md

```diff
@@ -26,7 +26,7 @@ exports.say = function () {
 }
 ```
 
-So if you run `var example = require('./example.js')`, then `example.js` will get evaluated and then `example` be an object equal to:
+So if you run `var example = require('./example.js')`, then `example.js` will get evaluated and then `example` will be an object equal to:
```

Comment:
```suggestion
So if you run `var example = require('./example.js')`, then `example.js` will get evaluated and then `example` becomes and object equal to:
```

Reply:
Thank you. I went with `becomes an` rather than `becomes and`.

c88eb7e5f9501ca7fe42968872135d93962373c2